README.md

```diff
@@ -21,12 +21,12 @@ Find current and past releases on the [releases page](https://github.com/Vonage/
 ### macOS
 
 ```
-curl -L https://raw.githubusercontent.com/Vonage/cloud-runtime-cli/main/script/install.sh | sh
+curl -s -L https://raw.githubusercontent.com/Vonage/cloud-runtime-cli/main/script/install.sh | sudo sh
```

Comment:
is the sudo needed? if you run it without sudo it will either go through or throw a permissions error no?